src/Utils/StyleAttributesUtils.php

```diff
@@ -327,6 +327,27 @@ public static function get_align_class_and_style( $attributes ) {
 		return null;
 	}
 
+
+	/**
+	 * If spacing value is in preset format, convert it to a CSS var. Else return same value
+	 * For example:
+	 * "var:preset|spacing|50" -> "var(--wp--preset--spacing--50)"
+	 * "50px" -> "50px"
+	 *
+	 * @param string $spacing_value value to be processed.
+	 *
+	 * @return (string)
+	 */
+	public static function get_spacing_value( $spacing_value ) {
+		// If value is in format: var:preset|spacing|50.
+		if ( is_string( $spacing_value ) && str_contains( $spacing_value, 'var:preset|spacing|' ) ) {
+			$spacing_value = str_replace( 'var:preset|spacing|', '', 'var:preset|spacing|50' );
+			return sprintf( 'var(--wp--preset--spacing--%s)', $spacing_value );
+		}
```

Comment:
Minor comment: I would add a reference to the code we used as a reference: https://github.com/WordPress/gutenberg/blob/cff6d70d6ff5a26e212958623dc3130569f95685/lib/block-supports/layout.php/#L219-L225.

Reply:
Done; added the reference in commit [a65dbb0](https://github.com/woocommerce/woocommerce-blocks/pull/7709/commits/a65dbb0f02f384b7b957ab3d31ffe01d346a9aaa). Merging with trunk now 🤝